app/index.html

```diff
@@ -408,8 +407,9 @@ <h1>JavaScript Required</h1>
         <script src="scripts/extensions/nav/dropdownMobile.js"></script>
         <!-- endbuild -->
 
-    <!-- Load any extensions last. -->
-    <script src="scripts/extensions.js"></script>
-
+        <!-- build:remove -->
```

Comment:
This is clever, seems like it will do the trick.

Reply:
Wonder if we should drop the `extensions` directory && load this file from `test/integration/extensions` or something.